docs/ansible.md

```diff
@@ -8,20 +8,36 @@ The inventory is composed of 3 groups:
 
 * **kube-node** : list of kubernetes nodes where the pods will run.
 * **kube-master** : list of servers where kubernetes master components (apiserver, scheduler, controller) will run.
-  Note: if you want the server to act both as master and node the server must be defined on both groups _kube-master_ and _kube-node_
+
+  Note: if you want the server to act both as master and node, the server must be defined on both groups _kube-master_ and _kube-node_.
+  If you want a standalone and unschedulable master, the server must be defined only in the _kube-master_ and not _kube-node_.
 * **etcd**: list of server to compose the etcd server. you should have at least 3 servers for failover purposes.
 
+  Note: do not place the _etcd_ group into the _k8s-cluster_, unless you have it fully contained in the latter:
+
+  ```
+  k8s-cluster ⊂ etcd => kube-node ∩ etcd = etcd
+  ```
+
+  When _kube-node_ contains _etcd_, you define your etcd cluster to be as well schedulable for Kubernetes workloads.
```

Comment:
This doesn't read so well. I would rather state it as:

If a node belongs to both groups _kube_node_ and _etcd, that means that this host will serve in the etcd cluster and be schedulable for Kubernetes workloads. This may impact performance under load. You may want to consider using standalone etcd nodes.